### PR TITLE
Adds timing functions

### PR DESCRIPTION
**Pull request by ahamilton55**
- Adds a Timing function that will take in a time in nanoseconds and do
  the proper conversions for Datadog. This removes the need to do
  conversion from the user to the library.
- Adds a TimingInMilliseconds similar to what Datadog has in its own
  library to keep with compatibility.

### CODE REVIEW

File: statsdog.go [hunk 1]
```diff
@@ -319,6 +319,34 @@ func Histogram(name string, value float64, tags []string, rate float64) error {
...
+	return c.send(name, stat, tags, rate)
+}
+
+// Timing sends a timing provided in nanoseconds as milliseconds
+// Helper to send in a timing since Go keeps timing data in nanoseconds and Datadog expects milliseconds
+func Timing(name string, value int, tags []string, rate float64) error {
+	timeInMs := float64(value / 1000000.0)
+	stat := strconv.FormatFloat(timeInMs, 'f', 6, 64) + "|ms"
+	return std.send(name, stat, tags, rate)
+}
```
Comment (steven-liu) [inline statsdog.go]: Since `std` is an instance of the `Client` struct, should we just call `std.Timing(name, value, tags, rate)`?

Comment (ahamilton55) [inline statsdog.go]: I was just following the convention for the rest of the functions already in this file. I can change this though.

**Comment by steven-liu** [comment]
Not related to PR content, but should we move instantiation of `std` into an `init()` method? or does it not matter?

**Comment by steven-liu** [comment]
aside from my comments above, LGTM :+1:.

**Comment by ahamilton55** [comment]
@steven-liu I've moved everything to just use the TimeInMilliseconds that is part of the client since all of these are really doing the same thing just with a conversion in the Timing method.

**Comment by steven-liu** [comment]
sounds good.

**Comment by KyleJamesWalker** [comment]
LGTM :8ball:

**Pull request merged**